### PR TITLE
Mobile version with a contact form

### PR DESCRIPTION
**Pull request by cmolinan**
### Changes
* A contact form was added to the mobile version of the portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,6 +221,42 @@ <h3>Skills</h3>
...
+              <li>
+                <textarea placeholder="Write your message here"
+                 class="txt" name="contact_text" 
+                 cols="30" rows="5"  maxlength="500" required
+                 ></textarea>
+              </li>
+              <li>
+                <button type="submit" class="contact-button">
+                  Get in Touch
+                </button>
```
[the remark this answers is not in the document]
Comment (zieeco) [reply index.html:253]: *README FILE*

- Please, kindly use the [microverse readme template](https://github.com/microverseinc/readme-template) to update your readme file by doing the following to make your readme more professional below:

   -  Please, kindly add a getting started header and add some steps on how users can have your repo running locally on their machine.
   - Please, kindly add the "Acknowledgment" header and add its content.
   - Please, kindly add the "Contributing" header and add its content.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @cmolinan,


Great job so far! 👏 
You've done well implementing the contact form, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ismailco) [inline index.html:253]: - [ ] As the previous reviewer muntioned, kindly implement Button interactions such as hover, and active (when pressed), so they are styled as shown in the Figma template.

btn

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @cmolinan, @LokurasRlz, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @cmolinan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**